config/default.yml

```diff
@@ -333,6 +333,14 @@ Rails/Present:
   # Convert usages of `unless blank?` to `if present?`
   UnlessBlank: true
 
+Rails/RakeEnvironment:
+  Description: 'Set :environment task as a dependency to all rake task.'
+  Enabled: true
+  VersionAdded: '2.4'
```

Comment:
Probably not a problem in most cases, but it is likely that calling `:environment` task will break a behavior. It may be better to mark it as `Safe: false`. WDYT?

Reply:
I agree with you. I added `Safe: false`. Thanks!